lib/Capabilities/CapabilitiesDataType.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { MultiColumnList, Headline } from '@folio/stripes/components';
```

Comment:
Please re-order and group together third-party, stripes and local imports across the new files

Reply:
done